Tests/Runner/BranchedWorkspaceTest.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+namespace Keboola\DockerBundle\Tests\Runner;
+
+use Keboola\Csv\CsvFile;
+use Keboola\DockerBundle\Docker\Component;
+use Keboola\DockerBundle\Docker\JobDefinition;
+use Keboola\DockerBundle\Docker\Runner;
+use Keboola\DockerBundle\Docker\Runner\UsageFile\NullUsageFile;
+use Keboola\DockerBundle\Tests\BaseRunnerTest;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\Components;
+use Keboola\StorageApi\DevBranches;
+use Keboola\StorageApi\Options\Components\Configuration;
+use Keboola\StorageApiBranch\ClientWrapper;
+use Keboola\Temp\Temp;
+
+class BranchedWorkspaceTest extends BaseRunnerTest
```

Comment:
V ramci seznamovani s tim, jak vse funguje, jsem tenhle test napsal v podstate na cisto (postupnym prebiranim malych kousku z jinych testu). @odinuv byl bych rad, pokud bysme si to mohli nejak vic v detailu projit, jestli je to v pohode, protoze to vypada o dost jinak, nez ostatni testy v projektu.